examples/window-app/silabs/include/WindowManager.h

```diff
@@ -160,7 +160,8 @@ class WindowManager
 
     LEDWidget mStatusLED;
     LEDWidget mActionLED;
-
+    // Get QR Code and emulate its content using NFC tag
+    char mQRCodeBuffer[chip::QRCodeBasicSetupPayloadGenerator::kMaxQRCodeBase38RepresentationLength + 1];
```

Comment:
This doesn't seem to match how our other applications work.

Reply:
Removed this as it is no longer needed.